tensorboard/uploader/uploader_test.py

```diff
@@ -311,7 +314,8 @@ def test_start_uploading_graphs(self):
         for (i, call) in enumerate(mock_client.WriteBlob.call_args_list):
             requests = list(call[0][0])
             data = b"".join(r.data for r in requests)
-            self.assertEqual(data, graph_event.graph_def)
+            actual_graph_def = graph_pb2.GraphDef.FromString(data)
+            self.assertEqual(actual_graph_def, expected_graph_def)
```

Comment:
might as well use `assertProtoEquals()` for the nicer diff?

Reply:
Sure; done.

Reply:
I can never remember the `assertProtoEquals` argument order; thankfully,
a web search turns up familiar faces…

![Screenshot of DuckDuckGo search for `assertprotoequals`: the top
result is a TensorBoard commit!][ss]

[ss]: https://user-images.githubusercontent.com/4317806/79261133-3d53cf00-7e44-11ea-930e-71e81956f3b6.png